soc/xtensa/intel_adsp/common/clk.c

```diff
@@ -78,12 +78,12 @@ void cavs_clock_init(void)
 	uint32_t platform_lowest_freq_idx = CAVS_CLOCK_FREQ_LOWEST;
 	int i;
 
-#ifdef CONFIG_SOC_SERIES_INTEL_CAVS_V25
+#ifdef CONFIG_HAS_WOV_CRO
 	CAVS_SHIM.clkctl |= CAVS_CLKCTL_WOVCRO;
 	if (CAVS_SHIM.clkctl & CAVS_CLKCTL_WOVCRO)
 		CAVS_SHIM.clkctl = CAVS_SHIM.clkctl & ~CAVS_CLKCTL_WOVCRO;
 	else
-		platform_lowest_freq_idx = CAVS_CLOCK_FREQ_WOVCRO;
+		platform_lowest_freq_idx = CAVS_CLOCK_FREQ_LPRO;
 #endif
 
 	for (i = 0; i < CONFIG_MP_NUM_CPUS; i++) {
```

Comment:
Is this loop supposed to run when using `CONFIG_HAS_WOV_CRO` ? Note that before this change this loop was only running when `(CAVS_SHIM.clkctl & CAVS_CLKCTL_WOVCRO)` was false.

Reply:
I'm not seeing how this change affects the loop, as it is conditioned by `CONFIG_MP_NUM_CPUS`. Am I missing something?

Reply:
weird ... somehow for me just appeared the deletion and not the line that was added.